framework/include/cppmicroservices/Any.h

```diff
@@ -139,6 +139,12 @@ namespace cppmicroservices
     template <typename ValueType>
     ValueType* any_cast(Any* operand);
```

Comment:
you can probably remove this forward declaratoin since it's no longer being called in `operator==`

Reply:
Removed.